config/webhook/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
+  - webhook.yaml
```

Comment:
can we remove the file "`manifests.yaml`" from **config/webhook** folder in case it is not used anywhere

Reply:
sure